src/domain/orchard_domain_vanilla.rs

```diff
@@ -4,6 +4,13 @@
 use blake2b_simd::Hash as Blake2bHash;
 use zcash_note_encryption_zsa::note_bytes::NoteBytesData;
 
+use super::{
+    orchard_domain::OrchardDomainCommon,
+    zcash_note_encryption_domain::{
+        build_base_note_plaintext_bytes, Memo, COMPACT_NOTE_SIZE_VANILLA, NOTE_VERSION_BYTE_V2,
+    },
+};
```

Comment:
Should not move existing imports

Reply:
1.	Preserve Import Order
Add this to your rustfmt.toml:
```toml
reorder_imports = false
```
This prevents rustfmt from reordering imports.
(this is needed accross everiting we do)

Reply:
Done